web-common/src/features/dashboards/url-state/url-state-variations.spec.ts

```diff
@@ -580,6 +586,7 @@ export function getCleanMetricsExploreForAssertion() {
   // TODO
   delete cleanedState.selectedScrubRange;
   delete cleanedState.leaderboardContextColumn;
+  delete cleanedState.leaderboardMeasureCount;
```

Comment:
We should not be ignoring this. I see tests fail if this is removed.

`selectedScrubRange` is renamed but no tests are added for it yet, same for `leaderboardContextColumn`.